source/_integrations/sql.markdown

```diff
@@ -79,117 +38,56 @@ In this section, you find some real-life examples of how to use this sensor.
 
 This example shows the previously *recorded* state of the sensor `sensor.temperature_in`.
 
-```yaml
-sensor:
-  - platform: random
-    name: Temperature in
-    unit_of_measurement: "°C"
-```
-
 The query will look like this:
 
 ```sql
 SELECT * FROM states WHERE entity_id = 'sensor.temperature_in' ORDER BY state_id DESC LIMIT 1;
 ```
 
-```yaml
-# Example configuration.yaml
-sensor:
-  - platform: sql
-    queries:
-      - name: Temperature in
-        query: "SELECT * FROM states WHERE entity_id = 'sensor.temperature_in' ORDER BY state_id DESC LIMIT 1;"
-        column: "state"
-```
-
-Note that the SQL sensor state corresponds to the last row of the SQL result set.
+Use `state` as column for value.
```

Comment:
I would provide a screenshot *or* description of what goes in where in the config flow

Reply:
>  I would provide a screenshot or description of what goes in where in the config flow

We generally don't provide screenshots of configuration flows.